_posts/2019-04-26-workshop-rails-upgrade.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: "Workshop - Upgrade Rails 101: The Roadmap to Smooth Upgrades"
+date: 2019-04-05 10:00:00
+categories: ["upgrade-rails", "workshop"]
+author: "luciano"
+---
+
+Next week the [Rails](https://rubyonrails.org/) community is having the biggest event of the year, the [Rails Conf 2019](https://railsconf.com/). We participated on previous editions of this conference but this time we have the honor to be in the speakers side of it. We'll be teaching you what we're best at: [Rails Upgrades](https://fastruby.io/blog/tags/upgrades).
+
+<!--more-->
+
+<img src="/blog/assets/images/workshop-rails-conf.png" alt="Workshop Rails Conf">
+
+At [Fast Ruby](https://fastruby.io/) we have been doing Rails Upgrades for many years. That allowed us to work on all kind of projects. From tiny [MVP's](https://fastruby.io/minimum-viable-product) to huge [monolith](https://fastruby.io/monolith) applications.
+
+During the 2-hour [workshop](https://railsconf.com/program/workshops#session-776) you will learn a proven Rails upgrade process for major and minor version changes of Rails.
+We will teach you how you can [evaluate](https://fastruby.io/blog/upgrade-rails/legacy-rails-silently-judging-you.html) a Rails application before consider upgrading it. What tools you should us to make the upgrade easier. What are the main things that you should address on every upgrade. And much more.
```

Comment:
"before considerING upgrading it. What tools you should usE to make the upgrade easier. What THE MAIN THINGS ARE that you should address one every upgrade."